hoymiles_modbus/client.py

```diff
@@ -19,18 +21,14 @@
 class CommunicationParams:
     """Low level pymodbus communication parameters."""
 
-    timeout: int = 3
+    timeout: float = 3.0
     """Request timeout."""
     retries: int = 3
     """Max number of retries per request."""
-    retry_on_empty: bool = False
```

Comment:
I'm concerned about the interface change. This will cause breaking changes both in hoymiles_modbus and hoymiles_mqtt. I assume there is not much that can be done.

Reply:
Yes, there is an interface change.
Old interface isn't available, but there isn't much that can be done to it. There's even more changes when moving out of legacy framer.

Reply:
Some users were using these parameters to improve communication stability with their DTUs. So this may cause issues for them. I can highlight this in the release notes.